utils/fhir-server-export.py

```diff
@@ -0,0 +1,91 @@
+#!/usr/bin/env python3
+"""Util to download all resources from the given FHIR server"""
+import requests, sys, time
+
+
+def download_file(url, filename=None):
+    """Download given large file via streaming"""
+    # https://stackoverflow.com/a/16696317
+    if not filename:
+        filename = url.split("/")[-1]
+    with requests.get(url, stream=True) as r:
+        r.raise_for_status()
+        with open(filename, "wb") as f:
+            for chunk in r.iter_content(chunk_size=8192):
+                # If you have chunk encoded response uncomment if
+                # and set chunk_size parameter to None.
+                # if chunk:
+                f.write(chunk)
+    return filename
+
+
+def poll_status(status_poll_url):
+    """Poll given status URL until ready (or timeout). Returns response JSON when ready to download"""
+    # TODO set via argparse
+    # in seconds
+    max_rety_time = 10 * 60
+    rety_time = 0
+
+    while rety_time < max_rety_time:
+        status_poll_response = requests.get(status_poll_url)
+        status_poll_response.raise_for_status()
+
+        retry_after = int(status_poll_response.headers.get("Retry-After", 0))
+        if not retry_after:
+            return status_poll_response.json()
+
+        progress = status_poll_response.headers.get("X-Progress")
+        if progress:
+            print("progress: ", progress)
+
+        print(f"waiting {retry_after} seconds")
+        rety_time += retry_after
+        time.sleep(retry_after)
```

Comment:
```suggestion
    max_retry_time = 10 * 60
    retry_time = 0

    while retry_time < max_retry_time:
        status_poll_response = requests.get(status_poll_url)
        status_poll_response.raise_for_status()

        retry_after = int(status_poll_response.headers.get("Retry-After", 0))
        if not retry_after:
            return status_poll_response.json()

        progress = status_poll_response.headers.get("X-Progress")
        if progress:
            print("progress: ", progress)

        print(f"waiting {retry_after} seconds")
        retry_time += retry_after
        time.sleep(retry_after)
```

Reply:
are there changes in this diff? I keep squinting at it, but nothing's standing out to me

Reply:
Not that I can tell, but I think your question was for @daniellrgn